geode-protobuf/src/main/proto/basicTypes.proto

```diff
@@ -52,7 +52,12 @@ message CallbackArguments {
 
 message Region {
     string name = 1;
-    // TODO: key, value types?
+    string type = 2;
```

Comment:
While I don't really like the name "dataPolicy" it seems like renaming it could cause confusion.  Geode docs don't mention a Region "type".